include/structures/cluster.hpp

```diff
@@ -43,6 +43,9 @@ class Cluster
 
     // \brief Returns the average length of the inserted sequences of all cluster members.
     int32_t get_average_inserted_sequence_size() const;
+
+    // \brief Returns the members of the cluster.
+    std::vector<Junction> get_members() const;
```

Comment:
Is your goal to have a compatibility to doxygen?

```suggestion
    //! \brief Returns the members of the cluster.
    std::vector<Junction> get_members() const;
```